mongoclient/get_messages.go

```diff
@@ -47,7 +52,7 @@ func GetMessagesV2(ctx context.Context, topic string, from int64, limit int64, c
 			"timestamp": bson.M{
 				"$lte": from, // less than or equal
 			},
-			"blocked": false,
+			"blocked": retrieve,
```

Comment:
I believe that `retrieve` is not a good name to this param, something like `showBlocked` or `isBlocked` maybe fits better.

Reply:
[Done](https://github.com/topfreegames/mqtt-history/pull/20/commits/a551eaf2a8ab4649e414788fc480663499206200) 👍🏾